.semaphore/semaphore.yml

```diff
@@ -78,7 +78,7 @@ blocks:
               value: extensions-contrib/confluent-extensions
           commands: &run_tests
             - >
-              MAVEN_OPTS="${MAVEN_OPTS} -Xmx800m" ${MVN} test -pl ${MAVEN_PROJECTS}
+              MAVEN_OPTS="${MAVEN_OPTS} -Xmx1g" ${MVN} test -pl ${MAVEN_PROJECTS}
```

Comment:
@xvrl  I had to give more heap space for the tests to pass in 0.21-rc branch. What additional checks should I be doing on this branch given this increased memory requirements?

Reply:
link to semaphore build with GC issues: https://confluentinc.semaphoreci.com/jobs/0b9f8229-428d-4147-a40a-888820669726